src/lib/api/stock-news.ts

```diff
@@ -23,7 +23,7 @@ export async function getNewsInfoFromStockNews(): Promise<NewsInfo[]> {
   const params = new URLSearchParams({
     token: import.meta.env.VITE_STOCKNEWS_API_KEY,
     section: 'alltickers',
-    items: '3',
+    items: '100',
```

Comment:
Vi bör inte hämta 100 stycken på en och samma gång.

```suggestion
    items: '12',
```